app/parse.py

```diff
@@ -8,8 +18,66 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s] - [%(levelname)8s]: %(message)s",
+    handlers=[
+        logging.StreamHandler(sys.stdout),
+        logging.FileHandler("parser.log"),
+    ]
+)
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    text = quote_soup.select_one(".text").text
+    author = quote_soup.select_one(".author").text
+    tags = (
+        quote_soup.select_one(".tags").text
+        .replace(" ", "").split("\n")
+    )
+    tags = [tag for tag in tags if tag and tag != "Tags:"]
+    return Quote(text, author, tags)
+
+
+def get_num_pages(page_soup: BeautifulSoup) -> int:
+    page_soup.select_one()
```

Comment:
The `get_num_pages` function is incomplete. It should return the number of pages available for scraping. Consider using a selector to find the pagination element and extract the total number of pages.